Marlin/src/module/probe.cpp

```diff
@@ -109,9 +109,27 @@ float zprobe_zoffset; // Initialized by settings.load()
   void run_deploy_moves_script() {
     #ifndef TOUCH_MI_DEPLOY_XPOS
       #define TOUCH_MI_DEPLOY_XPOS 0
-    #elif X_HOME_DIR > 0 && TOUCH_MI_DEPLOY_XPOS > X_MAX_BED
+    #elif TOUCH_MI_DEPLOY_XPOS > X_MAX_BED
       TemporaryGlobalEndstopsState unlock_x(false);
     #endif
+
+    #if ENABLED(TOUCH_MI_MANUAL_DEPLOY)
+      const screenFunc_t prev_screen = ui.currentScreen;
+      PGM_P const touchmi_str = PSTR(MSG_MANUAL_DEPLOY_TOUCHMI);
+      ui.return_to_status(); // To display the new status message
+      ui.set_status_P(touchmi_str, 99);
```

Comment:
"Deploy TouchMI" is in the imperative voice. This makes no sense as a status message. It is a command to the user.